sw/host/opentitanlib/src/io/jtag.rs

```diff
@@ -107,3 +110,12 @@ pub enum JtagTap {
     /// Lifecycle Controller's TAP.
     LcTap,
 }
+
+impl fmt::Display for JtagTap {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            JtagTap::RiscvTap => write!(f, "RiscvTap"),
+            JtagTap::LcTap => write!(f, "LcTap"),
+        }
+    }
+}
```

Comment:
Is there a reason you need `Display` for `JtagTap`?  It seems it is only used to create a string form for the error message.  However, `JtagTap` is a trivially small enum that implements `Copy` and `Debug`.  You could simply make it be the argument type in the error and use the Debug formatter in the error string.